src/events.py

```diff
@@ -75,10 +85,14 @@ def copy(self, **kw):
         _kw.update(kw)
         return self.__class__(**_kw)
 
-    def __cmp__(self, other):
-        if self.tick < other.tick: return -1
-        elif self.tick > other.tick: return 1
-        return 0
+    def __lt__(self, other):
+        return (super(Event, self).__lt__(other) or
+                (super(Event, self).__eq__(other) and
+                 self.channel < other.channel))
```

Comment:
This changes the logic instead of just porting to Python 3.  Intentional?

(Also, see above re: functools.total_ordering)